src/main/java/com/karl/http/HTTPServer.java

```diff
@@ -0,0 +1,33 @@
+package com.karl.http;
+
+import java.io.IOException;
+import com.karl.wrappers.WrappedServerSocket;
+import com.karl.wrappers.WrappedSocket;
+
+public class HTTPServer {
```

Comment:
This is almost identical to `EchoServer` aside from the `handler`- is there a way to get one server class to handle multiple `handlers`?

Reply:
Absolutely, this is something i want to do, this is just the first thing i settled on.